CHANGELOG.md

```diff
@@ -2,11 +2,14 @@
 
 ## [Unreleased]
 ### Added
+- Implemented anytia and pypi versioneers (for looking up the current upstream package version)
+- Added possibility to download latest build from Koji instead of building it
+- Added functional tests run from cron jobs on travis CI
```

Comment:
I agree that these changes are beneficial, but from user's point of view nothing actually changed.
And I always thought changelog is meant to be for users.
But I suppose keeping it won't do any harm and users can just ignore it ;-)